src/main/java/kitchenpos/domain/Menu.java

```diff
@@ -10,6 +10,33 @@ public class Menu {
     private Long menuGroupId;
     private List<MenuProduct> menuProducts;
 
+    public Menu() {
+        this(null, null, null, null, null);
+    }
+
```

Comment:
```suggestion
    public Menu() {
    }
```
생성자 체이닝을 사용하셨네요! 그런데 모두 null을 전달할 것이라면 위와 같이 작성해도 동일한 동작을 할 것 같아요. 이런 경우에도 생성자 체이닝을 사용할 필요가 있을까요?

Reply:
후추 말씀 듣고서 무릎을 탁! 쳤습니다

생각없이 저렇게 하고 있었네요 허허허허허